### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by rushemaa**
## Portfolio: Create a desktop version

In this fourth milestone I:

- Used media queries to create a desktop version

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -6,12 +6,21 @@ body {
...
   cursor: pointer;
 }
 
-@media only screen and (min-width: 375px) and (max-width: 768px) {
+.fonts {
+  font-family: sans-serif, "Poppins";
+  font-style: normal;
+}
+
+@media only screen and (min-width: 375px) and (max-width: 767px) {
```
Comment (GedeonTS) [inline styles.css:23]: - [ ] Kindly make sure your design is responsive, by making sure it doesn't break while we are still between 375 and 1440 px of width :recycle: 
__For this, you can use % and vw as units of width and center you work cords__
127 0 0 1_5500_index html (1)

File: index.html [hunk 2]
```diff
@@ -83,9 +96,10 @@ <h3 class="project-title">Tonic</h3>
         </div>
       </section>
       <!-- project 2 -->
-      <section class="grid-items other-projects">
+      <section class="grid-items other-projects fonts reverse">
         <img
           class="project-img"
+          id="project-2"
           src="images/projects/project-2.png"
           alt="Project-1"
```
Comment (GedeonTS) [inline index.html:104]: - [ ] Kindly make sure your project cards look like the Figma design as requested in the project requirements, by using the proportions in terms of width and height.

Your design:
Screenshot from 2023-02-24 14-09-22

Figma:

Screenshot from 2023-02-24 14-13-11

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @rushemaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good use of media queries :+1: 
- No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -499,3 +479,529 @@ a:hover {
...
+    display: flex;
+    flex-direction: column;
+    border-top-left-radius: 100px;
+    margin-top: -114px;
+    background-image: url("images/contact-form-background-shapes.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 89.6vw 102.86vh;
+    text-align: center;
+  }
```
Comment (zieeco) [inline styles.css:946]: *[OPTIONAL]*

- Hey, please, let's improve our design on the contact section by using the correct font family and reducing the width to match the design as much as we can. See the images below:


*Figma design*
Screenshot from 2023-02-24 17-18-50

*Your design*
Screenshot from 2023-02-24 17-23-48

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @rushemaa, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete! Please, kindly fix the issue pointed out before you merge.  After fixing the issue then you can merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
_Check the comments under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (rushemaa) [reply styles.css:946]: Thanks, I am going to fix it right now

**Pull request merged**